spec/shopify_transporter/exporters/magento/product_exporter_spec.rb

```diff
@@ -366,23 +395,342 @@ module Magento
+                    tags: [],
+                    type: 'simple',
+                    parent_id: '801',
+                    another_attribute: 'another_value',
+                    additional_attributes: {
+                      item: [
+                        {
+                          key: 'color',
+                          value: '22',
+                        },
+                        {
+                          key: 'size',
+                          value: '47',
+                        },
+                      ]
+                    },
+                  }
+                ).and_return(
+                  option1_name: 'Color',
+                  option1_value: 'White',
+                  option2_name: 'Size',
+                  option2_value: 'XS'
+                )
+
+                expect(catalog_product_list_response_body).to receive(:body).and_return(
+                  catalog_product_list_response: {
+                    store_view: {
+                      item: [
+                        {
+                          product_id: '802',
+                          type: 'simple',
+                        },
+                      ],
+                    },
+                  }
+                ).at_least(:once)
+
+                expect(soap_client).to receive(:call)
+                  .with(
+                    :catalog_product_info,
+                    product_id: '802',
+                    attributes: {
+                      'additional_attributes' => { item: ['color', 'size'] }
+                    }
+                  )
+                  .and_return(catalog_product_info_response_body)
+
+                expect(catalog_product_info_response_body).to receive(:body)
+                  .and_return(
+                    catalog_product_info_response: {
+                      info: {
+                        product_id: '802',
+                        another_attribute: 'another_value',
+                        additional_attributes: {
+                          item: [
+                            {
+                              key: 'color',
+                              value: '22',
+                            },
+                            {
+                              key: 'size',
+                              value: '47',
+                            },
+                          ]
+                        },
+                      },
+                    }
+                )
+
+                expected_result = {
+                  product_id: '802',
+                  type: 'simple',
+                  parent_id: '801',
+                  images: [],
+                  tags: [],
+                  another_attribute: 'another_value',
+                  inventory_quantity: 5,
+                  option1_name: 'Color',
+                  option1_value: 'White',
+                  option2_name: 'Size',
+                  option2_value: 'XS',
+                  additional_attributes: {
```

Comment:
any reason why this can't be something like:
(name could be better but you get the idea)
```yaml
option_details: [
    {},
    {}
]
```
instead of `additional_attributes -> item ->` ?

Reply:
The `additional_attributes` is just the raw soap result we get from the product info soap call, the keys we care about are the `option1_name`, `option1_value`, etc.

We can remove the additional_attributes, but I left it in in case we find some other use for it in the conversion steps.

Reply:
It looks like it's just adding extra nesting that makes the conversion step more complex. I'd just pull it up a few levels for ease of access and simplicity.